include/cudnn_frontend/graph_interface.h

```diff
@@ -1699,6 +1707,31 @@ Graph::layernorm(std::shared_ptr<Tensor_attributes> x,
     return {Y, MEAN, INV_VARIANCE};
 }
 
+inline std::array<std::shared_ptr<Tensor_attributes>, 3>
+Graph::adalayernorm(std::shared_ptr<Tensor_attributes> x,
+                    std::shared_ptr<Tensor_attributes> scale,
+                    std::shared_ptr<Tensor_attributes> bias,
+                    AdaLayernorm_attributes attributes) {
+    // Set outputs
+    auto Y = attributes.outputs[AdaLayernorm_attributes::output_names::Y] = output_tensor(attributes.name + "::Y");
+    std::shared_ptr<Tensor_attributes> MEAN                               = nullptr;
+    std::shared_ptr<Tensor_attributes> INV_VARIANCE                       = nullptr;
+    if (attributes.forward_phase == NormFwdPhase_t::TRAINING) {
+        MEAN = attributes.outputs[AdaLayernorm_attributes::output_names::MEAN] =
+            output_tensor(attributes.name + "::MEAN");
+        INV_VARIANCE = attributes.outputs[AdaLayernorm_attributes::output_names::INV_VARIANCE] =
+            output_tensor(attributes.name + "::INV_VARIANCE");
+    }
+    // Set inputs
+    attributes.inputs[AdaLayernorm_attributes::input_names::X]     = x;
+    attributes.inputs[AdaLayernorm_attributes::input_names::SCALE] = scale;
+    attributes.inputs[AdaLayernorm_attributes::input_names::BIAS]  = bias;
+
+    sub_nodes.emplace_back(std::make_unique<AdaLayerNormNode>(std::move(attributes), context));
+
+    return {Y, MEAN, INV_VARIANCE};
```

Comment:
```suggestion
    return {std::move(Y), std::move(MEAN), std::move(INV_VARIANCE)};
```
output_tensor are movable right? shared_ptrs definitely are.